lib/Adapter/Php/Serialized/FileRepository.php

```diff
@@ -48,7 +67,49 @@ public function put(Record $record): void
      */
     public function get(Record $record): ?Record
     {
-        return $this->deserializeRecord($record);
+        $bufferKey = $this->bufferKey($record);
+
+        if (isset($this->buffer[$bufferKey])) {
+            return $this->buffer[$bufferKey];
+        }
+        
+        $path = $this->pathFor($record);
+
+        if (!file_exists($path)) {
+            return null;
+        }
+        
+        $contents = file_get_contents($path);
+        
+        try {
+            $deserialized = @unserialize($contents);
+        } catch (CorruptedRecord $corruption) {
```

Comment:
This isn't a thing currently as it was removed in refactoring ... need to find another way to deal with this.